src/site/content/en/blog/mailru-cwv/index.md

```diff
@@ -0,0 +1,290 @@
+---
+layout: post
+title: Improving Core Web Vitals on the Mail.ru home page resulted in an average 10% increase in conversion rates
+subhead: |
+  Several months of work to improve Core Web Vitals (CWV) metrics on the home page of Mail.ru resulted in a 60% increase in the 75th percentile in Cumulative Layout Shift (CLS), boosting average session time by 2.7% and conversion rates of core sections by more than 10%.
+description: |
+  Several months of work to improve Core Web Vitals (CWV) metrics on the home page of Mail.ru resulted in a 60% increase in the 75th percentile in Cumulative Layout Shift (CLS), boosting average session time by 2.7% and conversion rates of core sections by more than 10%.
+authors:
+  - dstasyev
+  - svenmay
+date: 2021-11-15
```

Comment:
Is this the intended launch date, or was this padded in case the review needed a lot of time? Either way, feel free to adjust prior to merge.

Reply:
Happy to adjust, but I do think 2021-11-15 is quite likely :-)